calculus.py

```diff
@@ -1,7 +1,9 @@
 """
 This module implements different integration and root finding algorithms
 """
+import math
 from scipy import optimize
+from scipy.optimize import bisect
```

Comment:
why not just use it from the import above this is a double import

Reply:
Solved